jak-na-konflikty.md

```diff
@@ -0,0 +1,110 @@
+# Jak zvládat konflikty s kolegou?
+
+Cílem této příručky je pomoct ti zvládat konfliktní situace mezi tebou a ostatními Appliftery.
+
+V Kodexu Appliftera se můžeš dočíst, že řešení konfliktů je přirozenou součástí každé organizace. Proto je důležité, abys věděl, jak ke konfliktu přistupovat tak, abys minimalizoval jeho toxicitu pro tebe a pro tvého kolegu a zároveň aby tvé jednání směřovalo k jeho vyřešení.
+
+Níže najdeš doporučený postup, jak konflikty řešit tak, aby nám tu bylo hezky. Je jasné, že každý konflikt je trošku jiný a ne všechny lze řešit stejně. Pokud si nevíš rady, zajdi se poradit za radou starších.
+
+## Příprava
+
+Před tím, než začneš s kolegou konflikt řešit, je dobré si v hlavě či na papíře ujasnit pár věcí. 
+
+**Cílem přípravy není mít co nejvíc argumentů k tomu, abys svůj protějšek ubil argumenty.** Cílem je schopnost nahlédnout na danou věc z více úhlů, vžít se do svého kolegy a pochopit jeho motivaci a chování.
+
+- Co je věcným "odemočněným" předmětem konfliktu? Jak se daný problém dá popsat co nejvíc věcně?
+- Jak bude předmět konfliktu vnímat a vidět tvůj kolega?
+- Jaké kolem toho máš emoce? Jak se cítíš?
+- Jaké kolem toho bude mít emoce tvůj kolega? 
+- Jaké asi byly motivace kolegy? Proč postupoval tak jak postupoval?
+
+## Před konfliktem
+
+Pokud máš s nadcházejícím konfliktem spojených hodně negativních emocí, zkus je nejdříve zmírnit. 
+
+Věci, co ti můžou v tomhle pomoci:
+
+ * Sedni si sám v tiché místnosti na 5 minut a zamedituj si o tvém kolegovi. Soustřeď se na jeho kladné vlastnosti a hezké věci, co jste spolu zažili. Uvědom si, že nacházející konflikt je pravděpodobně něco, čemu se za půl roku nad pivem zasmějete.
+ * Jdi na chvíli pracovat nebo dělat něco jiného. Dělej něco, co ti dělá radost a obnoví tvoji víru v dobro lidí.
+
+
+## Vyvolání řešení konfliktu
+
+Zastav kolegu v soukromí a popros ho, jestli by si na tebe na chvilku neudělal čas. Řekni mu, že s ním potřebuješ vyřešit nějaké pnutí (klidně to můžeš říct přátelštější či vtipnější formou). Pokud čas nemá, domluv se na čase, kdy se to bude oběma stranám hodit.
+
+Pokud kolega nechce řešit konflikt s tebou o samotě, vyhledejte rovnou mediátora.
+
+## Konflikt ve skupině
+
+Může se stát, že má s kolegou konflikt celá skupina. Tato situace je velmi složitá ke korektnímu řešení a zejména velmi nekomfortní pro konfrontovaného kolegu. Měl by si se proto držet těchto pravidel:
+
+* Pokud máš s kolegou osobní konflikt, nezatahuj do něj další kolegy
+* Neřešte kolegovi věci za zády
+* Ber ohled na konfrontovaného, pokud budete řešit konflikt pod nátlakem, může se stát, že bude řešení neefektivní.
+
+Vyvolání komnfliktu za skupinu by měl dělat jeden zástupce a měl by kolegovi zvědomit, že se jedná a skupinový konflikt.
```

Comment:
Typo komnfliktu. Vlastně je tu docela hezké commonflikt :Joy:

Reply:
A měl by to dělat jeden? Mně to přijde takový blbý, protože to pak vede k tomu, že se mu řeší věci za zády. Spíš by si skupina rovnou měla pozvat mediátora.

Reply:
Aha, ono je to o řádek níž 😄. No tak potom není potřeba, aby to dělal jen jeden zástupce, ne?

Reply:
Mediátor se musí vybírat až s tím, s kým konflikt řešíš. Nesmí být předem domluven